lib/pbench/server/database/models/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+For any new database model added in the models directory
+an import statemnt of the same is required here.
```

Comment:
Does this really work?  We need to make sure the location where the database is supposed to be initialized does that, can't that be implemented instead in one function called in all the right places?

Reply:
So because all the models are imported here we can do `from pbench.server.database.models import *` in our database module's `__init__.py` file, that is where we invoke the `Database.init_db` method. Therefore before invoking the init_db call we indirectly import all the models as well

Reply:
s/statemnt/statement/

Reply:
@dbutenhof, what about #2255, which adds a module to the `models` directory -- will that require adding an import below, or do we need to rephrase the usage of "any", above?